packages/dds/matrix/src/test/memory/matrix.spec.ts

```diff
@@ -1,155 +0,0 @@
-/*!
- * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
- * Licensed under the MIT License.
- */
-
-import {
-	type IMemoryTestObject,
-	benchmarkMemory,
-	isInPerformanceTestingMode,
-} from "@fluid-tools/benchmark";
-import type { IChannel } from "@fluidframework/datastore-definitions/internal";
-import { MockFluidDataStoreRuntime } from "@fluidframework/test-runtime-utils/internal";
-
-import { SharedMatrix, type ISharedMatrix } from "../../index.js";
-import { matrixFactory } from "../utils.js";
-
-function createLocalMatrix(id: string): ISharedMatrix & IChannel {
-	return matrixFactory.create(new MockFluidDataStoreRuntime(), id);
-}
-
-describe("Matrix memory usage", () => {
-	// IMPORTANT: variables scoped to the test suite are a big problem for memory-profiling tests
-	// because they won't be out of scope when we garbage-collect between runs of the same test,
-	// and that will skew measurements. Tests should allocate all the memory they need using local
-	// variables scoped to the test function itself, so several iterations of a given test can
-	// measure from the same baseline (as much as possible).
-
-	beforeEach(async () => {
-		// CAREFUL: usually beforeEach/afterEach hooks are used to initialize or interact with variables
-		// whose scope is the encompasing test suite, but that's a problem for memory-profiling tests.
-		// See the comment at the top of the test suite for more details.
-	});
-
-	afterEach(() => {
-		// CAREFUL: usually beforeEach/afterEach hooks are used to initialize or interact with variables
-		// whose scope is the encompasing test suite, but that's a problem for memory-profiling tests.
-		// See the comment at the top of the test suite for more details.
-	});
```

Comment:
I think all the comments here are important. I see the new file has empty `beforeEach()` and `afterEach()` but they seem to be superfluous (why are they even there if they're empty?), unless we put these comments back. Let's restore them?

Reply:
I was suggesting we just remove the blocks. I'd be fine with this too, though. Either way.

Reply:
Removed